Makefile

```diff
@@ -152,6 +152,15 @@ else
 endif
 endif
 
+i18n-extract-server:
+	goi18n extract -format json -outdir assets/i18n/ server/
+	touch assets/i18n/translate.es.json
+	goi18n merge -format json -outdir assets/i18n/ assets/i18n/active.*.json assets/i18n/translate.es.json
+
+i18n-merge-server:
+	goi18n merge -format json -outdir assets/i18n/ assets/i18n/active.*.json assets/i18n/translate.es.json
+	rm assets/i18n/translate.es.json
```

Comment:
0/5 maybe put `assets/i18n/translate.*.*` into `gitignore`?

Reply:
I'll investigate `goi18n` tool for this too.

Reply:
Yes, I added them, and added an explanation in the make code to explain what to do after the `i18n-extract-server` execution.